### ISSUE TITLE
Problem with german MUI

### ISSUE THREAD

**Issue by felsgaertner** [iXab3r/MicSwitch#22]
On a German Windows 10 multi language UI the hotkeys are recognized as - in my example - *STRG+F12*:
![grafik](https://user-images.githubusercontent.com/46444829/108569630-7b473c80-730c-11eb-9142-65568bdfc77b.png)

This prevents a start of the application, the error log contains (line 3):
~~~text
2021-02-19 23:40:42,490 [HotkeyTracker] DEBUG Hotkey tracking disabled (hotkey gesture  => ) [PoeShared.UI.Hotkeys.HotkeyTracker]
2021-02-19 23:40:42,510 [HotkeyTracker] ERROR Unhandled application exception(CurrentDomainUnhandledException) [PoeShared.SharedLog]
System.NotSupportedException: Unbekanntes ModifierKeys-Element "STRG".
   bei System.Windows.Input.ModifierKeysConverter.GetModifierKeys(String modifiersToken, CultureInfo culture)
   bei System.Windows.Input.ModifierKeysConverter.ConvertFrom(ITypeDescriptorContext context, CultureInfo culture, Object source)
   bei PoeShared.UI.Hotkeys.HotkeyConverter.ConvertFrom(ITypeDescriptorContext context, CultureInfo culture, Object sourceRaw)
   bei PoeShared.UI.Hotkeys.HotkeyConverter.ConvertFromString(String source)
   bei MicSwitch.Services.ComplexHotkeyTracker.<>c__DisplayClass12_0.<Initialize>b__0()
   bei System.Reactive.ObserverBase`1.OnNext(T value)
   bei System.Reactive.Subjects.FastImmediateObserver`1.EnsureActive(Int32 count)
   bei System.Reactive.Subjects.ReplaySubject`1.ReplayBase.Subscribe(IObserver`1 observer)
   bei System.ObservableExtensions.SubscribeSafe[T](IObservable`1 source, IObserver`1 observer)
--- Ende der Stapelüberwachung vom vorhergehenden Ort, an dem die Ausnahme ausgelöst wurde ---
   bei System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   bei System.Reactive.Stubs.<>c.<.cctor>b__2_1(Exception ex)
   bei System.Reactive.ObserverBase`1.OnError(Exception error)
   bei System.ObservableExtensions.SubscribeSafe[T](IObservable`1 source, IObserver`1 observer)
   bei MicSwitch.Services.ComplexHotkeyTracker.Initialize()
   bei System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   bei System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   bei System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   bei System.Threading.ThreadHelper.ThreadStart(Object obj)
~~~

If I change the hotkey modifier to *CTRL* (like CTRL+F12), it starts and works:
From:
~~~text
          "MicrophoneHotkey": "STRG+F12",
~~~
To:
~~~text
          "MicrophoneHotkey": "CTRL+F12",
~~~

Please enable support for localized Windows MUI.

**Comment by felsgaertner** [comment]
I have to edit the config file every time before I run this program, since this program updates the config file on exit.
Therefore the STRG always gets back.

**Comment by iXab3r** [comment]
Thanks for the report ! Fixed in MicSwitch v1.0.163

**Comment by thunfischbrot** [comment]
Same issue here with version 1.0.191.0 to 1.0.211.0:
![image](https://user-images.githubusercontent.com/583358/132386214-649ef68c-f8d8-4b5d-abaf-ad71a7ac0c0f.png)
